packages/datepicker/src/lib/Datepicker.tsx

```diff
@@ -0,0 +1,29 @@
+import { TextField } from '@mui/material';
+import { DatePicker as MuiDatePicker, DatePickerProps as MuiDatePickerProps } from '@mui/x-date-pickers/DatePicker';
+import type { Dayjs } from 'dayjs';
+
+export type DatepickerProps = Omit<
+  MuiDatePickerProps<Dayjs, Dayjs>,
+  | 'components'
+  | 'componentsProps'
+  | 'desktopModeMediaQuery'
+  | 'DialogProps'
+  | 'OpenPickerButtonProps'
+  | 'openTo'
+  | 'orientation'
+  | 'PaperProps'
+  | 'PopperProps'
+  | 'reduceAnimations'
+  | 'renderInput'
+  | 'rifmFormatter'
+  | 'showToolbar'
+  | 'ToolbarComponent'
+  | 'toolbarFormat'
+  | 'toolbarPlaceholder'
+  | 'toolbarTitle'
+  | 'TransitionComponent'
+>;
+
+export const Datepicker = (props: DatepickerProps): JSX.Element => {
+  return <MuiDatePicker {...props} renderInput={(params) => <TextField {...params} />} />;
```

Comment:
Should we be touching renderInput? Wondering if we'll be able to pass custom props through slotProps once we have a custom textfield, or if we'll need users to handle renderInput to pass what they need directly to the TextField.